src/librustc_middle/mir/interpret/error.rs

```diff
@@ -556,6 +577,9 @@ impl dyn MachineStopType {
     }
 }
 
+#[cfg(target_arch = "x86_64")]
+static_assert_size!(InterpError<'_>, 40);
```

Comment:
Uff, 40 bytes. @oli-obk we might have to work on reducing that before @nnethercote catches us.^^